shared/constants/permissions.ts

```diff
@@ -24,6 +24,7 @@ export const RestrictedMethods = Object.freeze({
  */
 export const EndowmentPermissions = Object.freeze({
   'endowment:network-access': 'endowment:network-access',
+  'endowment:webassembly': 'endowment:webassembly',
```

Comment:
Looks like WebAssembly was already adde to this list in another PR

Reply:
Yes, it was added in the release PR. I removed it now.